### ISSUE TITLE
Allow for class autoloading (composer)

### ISSUE THREAD

**Issue by paxperscientiam** [monperrus/bibtexbrowser#68]
Re PR #67 

As bibtexbrowser can now be added to user projects via composer, the next logical step in this regard is to make bibtexbrowser compatible with composer's autoload.

Autoloading grants the user immediate access to namespaced classes of autoloaded libraries.

So, assuming you use a namespace of `Monperrus`, a user could do something like this:
````php
require_once 'path/to/composers/autoload.php';
use Monperrus\Bibtexbrowser as BibLib;

$bibLib = new BibLib;

$bibDb = $bibLib->load('path/to/mybib.bib');

$bibDb->manipulations();

echo $bibDb;
````

What do you think?

EDIT: fixed grammar

**Comment by monperrus** [comment]
Hi Chris, it looks like a really good idea.

**Comment by paxperscientiam** [comment]
So, I'll give it a go!

**Comment by monperrus** [comment]
closed per #63
